app/presenters/homepage_presenter.rb

```diff
@@ -0,0 +1,14 @@
+class HomepagePresenter < ContentItemPresenter
+  def links
```

Comment:
There's a pattern in the other presenters we should follow here. Something like:

```
PASS_THROUGH_KEYS = %i[
    links
  ].freeze

  PASS_THROUGH_KEYS.each do |key|
    define_method key do
      content_item[key.to_s]
    end
  end
```

Reply:
Done